### PR TITLE
hack deadline to fix timeout

### PR DESCRIPTION
**Pull request by typhoonzero**
Hack fix for https://github.com/PaddlePaddle/Paddle/issues/11710

### CODE REVIEW

File: paddle/fluid/operators/distributed/rpc_client.cc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #include "gflags/gflags.h"
 
 // default to 3min to avoid temprary network failures.
```
Comment (Yancey0623) [inline paddle/fluid/operators/distributed/rpc_client.cc:18]: Add a TODO comment here? and `3min` => `60min`

Comment (typhoonzero) [reply paddle/fluid/operators/distributed/rpc_client.cc:18]: Done

Comment (gongweibao) [reply paddle/fluid/operators/distributed/rpc_client.cc:18]: 这个。。。也。。。

**Review by Yancey0623 — APPROVED** [review]
LGTM.

**Review by panyx0718 — APPROVED** [review]

**Comment by panyx0718** [comment]
I restarted the CI

**Pull request merged**